### PR TITLE
Fedora one0one

### PR DESCRIPTION
**Pull request by munroesj52**
This fix the INC_AMINCLUDE issue

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 
-#@INC_AMINCLUDE@ is recommented for automake but does not work correcly.
-#So comment it out for now.
-#@INC_AMINCLUDE@
+#@INC_AMINCLUDE@ is recommented for automake and doxygen
+#but there is an naming comflict. So we rename aminclude.am to
+#doxygen.am from srcdir with allows INC_AMINCLUDE to generate 
+#aminclude.am to blddir.
+@INC_AMINCLUDE@
+include $(top_srcdir)/doxygen.am
```
Comment (gftg85) [inline Makefile.am]: The `@INC_AMINCLUDE` directive will cause configure to generate `aminclude.am` in the build dir.  That in itself is fine.  The only problem is that it causes `make dist` to fail when it tries to clean the working dir with `make dist clean`.

Adding the following line in this spot (and regenerating Makefile.in) will fix it:

```
distclean-local:
        rm $(top_builddir)/aminclude.am
```
(the eight spaces above need to be replaced with a tab, as required by make)

Comment (munroesj52) [reply Makefile.am]: Ok, the patch addresses this and implements the Fedora preference for separating licenses from doc.

Comment (gftg85) [reply Makefile.am]: Thanks.

**Comment by munroesj52** [comment]
Ok I pull/merged this change then taged v1.0.2

**Pull request merged**